src/xds/server.rs

```diff
@@ -173,8 +176,10 @@ impl ControlPlane {
             yield response;
 
             loop {
+                tracing::trace!("stream loop");
```

Comment:
Just double checking that you wanted to leave these `trace!` statements in here before merging.

Otherwise, merge when ready 👍🏻